edit-post/components/header/index.js

```diff
@@ -49,7 +53,10 @@ function Header( {
 			<HeaderToolbar />
 			{ ! isPublishSidebarOpen && (
 				<div className="edit-post-header__settings">
-					<PostSavedState />
+					<PostSavedState
+						forceIsDirty={ hasActiveMetaboxes }
```

Comment:
Minor: Curious if we need to distinguish this from the presentational `isDirty` prop, or if we ought to just pass this as `isDirty` and defer to use it in the `mapStateToProps`.

Reply:
you mean shortcut the `isDirty` selector entirely right? which would also mean add a check for post dirtiness here. I don't feel strongly either way.